meerk40t/kernel.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, context: "Context", name: str = None, channel: "Channel" = No
         self.context = context
         self.name = name
         self.state = STATE_INITIALIZE
+        self.kernel = self.context._kernel
+        self.root = self.kernel.root
```

Comment:
I'm pretty sure these exist as like property values or something. Like context.kernel is used a lot without the _kernel. It just does a property reference rather than an attribute like that.

Reply:
These are just conveniences. So you can say `self.root instead of `self.context._kernel.root`.